src/components/PeoplePage.tsx

```diff
@@ -1,29 +1,78 @@
-import { PeopleFilters } from './PeopleFilters';
-import { Loader } from './Loader';
-import { PeopleTable } from './PeopleTable';
+import { PeopleFilters } from '../components/PeopleFilters';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable';
+import { useEffect, useState } from 'react';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { getFilteredPeople } from '../utils/getFilteredPeople';
+import { useSearchParams } from 'react-router-dom';
 
 export const PeoplePage = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [people, setPeople] = useState<Person[]>([]);
+  const [error, setError] = useState('');
+
+  const [searchParams] = useSearchParams();
+
+  const loadPeople = () => {
+    setIsLoading(true);
+    setError('');
+
+    getPeople()
+      .then(res => {
+        const peopleWithFamily = res.map(person => {
+          const father = res.find(p => p.name === person.fatherName);
+          const mother = res.find(p => p.name === person.motherName);
+
+          return {
+            ...person,
+            father,
+            mother,
+          };
+        });
+
+        setPeople(peopleWithFamily);
+      })
+      .catch(() => setError('Something went wrong'))
```

Comment:
Consider providing more detailed error information to the user. Currently, the error message is generic ('Something went wrong'). If possible, include more specific error details to help users understand what went wrong.